grafana-plugin/src/pages/incidents/Incidents.tsx

```diff
@@ -428,23 +431,23 @@ class _IncidentsPage extends React.Component<IncidentsPageProps, IncidentsPageSt
   };
 
   renderBulkActions = () => {
-    const { selectedIncidentIds, affectedRows, isHorizontalScrolling } = this.state;
+    const { selectedIncidentIds, isHorizontalScrolling } = this.state;
     const { store, theme } = this.props;
 
     if (!store.alertGroupStore.bulkActions) {
       return null;
     }
 
-    const { results } = AlertGroupHelper.getAlertSearchResult(store.alertGroupStore);
-
     const hasSelected = selectedIncidentIds.length > 0;
-    const isLoading = LoaderHelper.isLoading(store.loaderStore, ActionKey.FETCH_INCIDENTS);
-    const hasInvalidatedAlert = Boolean(
```

Comment:
I believe this didn't add any value whatsoever, due to this we were showing "Out of date results" which didn't make sense as we didn't invalidate any data whatsoever